tests/component/component.test.ts

```diff
@@ -2670,6 +2670,56 @@ describe("other directives with t-component", () => {
     await nextTick(); // wait for changes triggered in mounted to be applied
     expect(fixture.innerHTML).toBe("<div><span>B0</span><span>B1</span></div>");
   });
+
+  test("t-on expression in t-foreach", async () => {
+    env.qweb.addTemplate("SomeWidget",
```

Comment:
this is a very old style of writing tests. I prefer using the `xml` tag helper instead. You most likely copied some code from an old test.